releases/templates/notes.txt

```diff
@@ -0,0 +1,20 @@
+[NOTES] Release team meeting {month}/{day}/{year}
+
+Hello everyone, here are the notes and highlights from today's Release team call. This is only an overview of the discussed items. If you'd like more info feel free to take a look at our notes and recording.
+
+* Topic A
+* Decision B
+
+Attendees:
+* A
```

Comment:
nit
```suggestion
* Name
```

Reply:
Indeed, even using A in both places can be misleading